.github/workflows/autopilot-branch-to-latest.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
     - name: Build Tool
       run: |
         cd hack/tool
-        make
+        make --jobs=$(nproc)
```

Comment:
I don't really understand how here and everywhere in the diff -j helps to parallelize jobs, majority of the targets are single external commands and all of the make calls uses only one target.

Reply:
This is to ensure that k0s actually supports parallel builds, not so much to speed up things. Parallel builds are advertised in the README, so we should ensure that there are no regressions.

It's so easy to hack on the Makefiles and do something that works flawlessly in sequential builds, but will break when run in parallel.

Reply:
Other option is to change the readme and drop the claim, because in fact we do not support parallel build more than go build toolchain supports it. Makefile in our case basically just a job runner, not the build tool per se. Claiming that we support parallel build by passing -j is not something that exactly reflects the reality.

Reply:
It actually speeds things up a bit when building the embedded binaries. We can also drop the support for parallel builds, but then I think we should explicitly say that in the README.

@ncopa: You added the note about parallel builds. Any thoughts on that?

Reply:
15-20% is a significant difference. We could also keep the -j for the embedded binaries call (I guess it's coming from `make k0s`?) but drop the claim and don't put -j everywhere.

Reply:
I think that ensuring support for parallel builds will also help us make sure that partial builds always works. I think its worth the extra complexity.